developers/weaviate/client-libraries/python/v3_v4_migration.md

```diff
@@ -96,6 +102,8 @@ To configure connection timeout values, see [Timeout values](/developers/weaviat
 </TabItem>
 </Tabs>
 
+Note that the `Client` object used in the `v3` API is [still available](./index.md#python-client-v3-api), but may be deprecated in the future.
```

Comment:
Consider:
The `v3` style `Client` object is [still available](./index.md#python-client-v3-api), but may be deprecated in the future.

or

Note: The `v3` style `Client` object is [still available](./index.md#python-client-v3-api), but may be deprecated in the future.

Reply:
I would even go as far as "and will be deprecated in the future" :p

Reply:
Updated